src/python/elements.cpp

```diff
@@ -135,14 +135,19 @@ namespace
 
 void init_elements(py::module& m)
 {
-    py::module_ const me = m.def_submodule(
+    py::module_ me = m.def_submodule(
         "elements",
         "Accelerator lattice elements in ImpactX"
     );
 
     // mixin classes
 
-    py::class_<elements::Named>(me, "Named")
+    py::module_ const mx = me.def_submodule(
+        "mixin",
+        "Mixin classes for accelerator lattice elements in ImpactX"
+    );
+
+    py::class_<elements::Named>(mx, "Named")
```

Comment:
@proy30 just FYI: these four elements should not be selectable by the user in the GUI. This change should do that automatically now.